Dynatrace/dynatrace-operator
# Makes ruxitagentproc.conf read/writeable by all users

# Description

Makes it so that the `ruxitagentproc.conf` has filemode 666 (read/write for all)

## How can this be tested?
Deploy the CSI driver (on a cluster whose nodes doesn't have any previous installs)
Deploy Dynakube with codeModules image or something that uses the CSI driver
Check if in the sample app everyone has read/write rights `/opt/dynatrace/oneagent-paas/agent/conf/ruxitagentproc.conf` file 

## Checklist
